src/providers/wfs/qgswfsdataitems.cpp

```diff
@@ -50,6 +55,76 @@ QgsWfsLayerItem::~QgsWfsLayerItem()
 {
 }
 
+QList<QMenu *> QgsWfsLayerItem::menus()
+{
+  QList<QMenu *> menus;
+
+  if ( mPath.startsWith( QLatin1String( "geonode:/" ) ) )
+  {
+    QAction *actionCopyStyle = new QAction( tr( "Copy Style" ), this );
+    connect( actionCopyStyle, &QAction::triggered, this, &QgsWfsLayerItem::copyStyle );
+
+    QMenu *menuStyleManager = new QMenu();
+    menuStyleManager->setTitle( tr( "Styles" ) );
+    menuStyleManager->addAction( actionCopyStyle );
+    menus << menuStyleManager;
+  }
+
+  return menus;
+}
+
+void QgsWfsLayerItem::copyStyle()
+{
+  QString layerUri = this->uri();
+
+  QgsGeoNodeConnection *connection = nullptr;
+  Q_FOREACH ( const QString &connName, QgsGeoNodeConnectionUtils::connectionList() )
```

Comment:
const QStringList connections = QgsGeoNodeConnectionUtils::connectionList();
for ( const QString &connName : connections )
 ...

Reply:
FYI @myarjunar https://github.com/qgis/QGIS-Enhancement-Proposals/issues/103